Implemented Spacer and Clef popups

anatoly-os opened the pull request:
- Implemented view and model of the spacer popup
- Implemented view and model of the clef popup

anatoly-os committed:
Implemented Spacer popup

Implemented view and model of the spacer popup.

anatoly-os commented:
Added a comment in the related issue: https://musescore.atlassian.net/browse/IN-25?focusedCommentId=10002.

Following points should be addressed:

- [x] Checkbox string state “Show courtesy clef on previous system”. In fact this checkbox affects the courtesy shown when changing a clef in the middle of a staff as well (https://github.com/vpereverzev/MuseScore/pull/12/files#diff-76cb5533afaf2080fc1b9248947ddb73R24)
- [x] Changing the option of the clef placed on a next system/page, the courtesy clef visibility isn’t affected (the list of elements is probably incorrect) (fixed: https://github.com/vpereverzev/MuseScore/pull/12/files#diff-f945e361977b127996e8edc4ae4b533cR280)

vpereverzev commented on inspectors/models/notation/clefs/clefsettingsmodel.h:
```diff
@@ -0,0 +1,26 @@
+{
+    Q_OBJECT
+
+    Q_PROPERTY(PropertyItem* showCourtesy READ showCourtesy CONSTANT)
```
Usually, I prefer to name the bool properties using markers "is" and "has" - that's a popular practice within any language

In QML, "model.showCourtesy" may sound like a direct action - i.e. "Q_INVOKABLE void showCourtesy()"

vpereverzev commented on inspectors/services/elementrepositoryservice.cpp:
```diff
@@ -40,6 +42,7 @@ QList<Ms::Element*> ElementRepositoryService::findElementsByType(const Ms::Eleme
     case Ms::ElementType::STAFF: return findStaffs();
     case Ms::ElementType::LAYOUT_BREAK: return findSectionBreaks(); //Page breaks and line breaks are of type LAYOUT_BREAK, but they don't appear in the inspector for now.
     case Ms::ElementType::PEDAL: return findPedals();
+    case Ms::ElementType::CLEF: return findCourtesyClefs(); //"Show Courtesy clef" property can be applied only to courtesy clefs
```
I would recommend transferring this comment to the model, where this property is used

and here I would write something like this - "Within the inspector, we use only this type of clefs, see the details in such a model"

anatoly-os committed:
Implemented Clef popup

- Implemented view and model of the spacer popup
- Removed obsolete ids from SectionBreak qml code
1

anatoly-os committed:
Applied better naming convention to Clef and SectionBreak models data members
